### ISSUE TITLE
Timestamps are not updated

### ISSUE THREAD

**Issue by 610th** [emdgroup/awscli-s3touch#6]
I'm trying to use the plugin as suggested in this Stack Overflow post: https://stackoverflow.com/a/50396550/6478211

`aws s3 touch my-bucket --prefix myfolder/`

**Expected outcome:** files located in `myfolder` have their timestamps updated to current date. 
**Actual outcome:** nothing happens. 

I will gladly provide more information if you ask. Right now I'm not sure about what to provide.

s3touch version: `master: fef8f40edc457b945139b14bb2c88bde66b2f3d7`
aws-cli version: `aws-cli/1.16.300 Python/3.6.9 Linux/5.0.0-37-generic botocore/1.13.36`

**Comment by adyekjaer** [comment]
Hi there, 

That is not the intention of this plugin. This plugin simulates a write of a file to S3 to trigger an event that other AWS services can subscribe to. 

The s3 object is left untouched.

The answer in that StackOverflow discussion is incorrect AFAIK.

~Anders

**Comment by 610th** [comment]
@adyekjaer Alright. Thanks. That's bad news for me. I got that the intention was different from reading the README, but the Stack Overflow post seemed to be posted by the maintainer of this repo.  

I guess this issue can be closed if it's not possible to do what I want.

**Comment by adyekjaer** [comment]
Right, well usually the need is to replay events not to actually change the timestamp. I'm pretty sure this requires you to write a bit of code that copy/pastes the object. You can if need be select an object or a full "directory" in the S3 console and select copy, then paste.

Cheers
